code/modules/organs/internal/augment.dm

```diff
@@ -145,6 +146,12 @@
 
 	var/mob/living/carbon/human/M = src
 
+	if(buckled)
+		var/obj/Ob = buckled
+		if(Ob.buckle_lying)
+			to_chat(M, "<span class='notice'>You are unable to enable your augments when restrained.</span>")
```

Comment:
`You cannot use your augments when restrained.`
`unable to enable` can be confusing